src/QuickMarcEditor/QuickMarcEditWrapper.test.js

```diff
@@ -312,14 +310,6 @@ describe('Given QuickMarcEditWrapper', () => {
         expect(mutator.quickMarcEditMarcRecord.PUT).toHaveBeenCalled();
 
         waitFor(() => expect(mockShowCallout).toHaveBeenCalledWith({ messageId: 'ui-quick-marc.record.save.success.processing' }));
-
-        await new Promise(resolve => {
```

Comment:
We should still keep this test - it's checking that if we're clicking on `Save & Close` button `onClose` is called
And there should be a different test case checking that when we click on `Save & keep editing` - `onClose` is not called